client/state/selectors/is-selected-site-domain-only-site.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { isDomainOnlySite, getSelectedSiteId } from 'state/selectors';
+
+/**
+ * Returns true if selected site is a Domain-only site, false if the site is a regular site,
+ * or null if there is no selected or site is unknown.
```

Comment:
My brain probably thinks that a subject is missing: `or null if there is no [site] selected or site is unknown`. Now that I read it again, maybe we don't have to be that specific:

```
Returns true if selected site is a Domain-only site, false if the site is a regular site, or null otherwise
```